### PR TITLE
chore: add ad for yuque

### PR DESCRIPTION
**Pull request by benjycui**
screen shot 2018-04-21 at 3 43 35 pm
screen shot 2018-04-21 at 3 43 32 pm

### CODE REVIEW

**Comment by benjycui** [comment]
deploy/netlify 报错是怎么回事？

File: site/theme/static/header.less [hunk 1]
```diff
@@ -142,3 +142,25 @@
     right: 16px;
   }
 }
+
+.promote-banner {
+  position: relative;
+  display: block;
```
Comment (afc163) [inline site/theme/static/header.less]: 要填底色的，否则超大屏幕两边会变成白色。

File: site/theme/template/Layout/index.jsx [hunk 2]
```diff
@@ -47,6 +50,10 @@ export default class Layout extends React.Component {
     this.state = {
       appLocale,
       isMobile,
+      adBannerClosed: (
+        window.localStorage &&
+          window.localStorage.getItem(`adBannerClosed-${promoteBannerImageUrl}`) === 'true'
```
Comment (afc163) [inline site/theme/template/Layout/index.jsx]: 服务端渲染不能用 window

Comment (afc163) [reply site/theme/template/Layout/index.jsx]: https://app.netlify.com/sites/ant-design/deploys/5adaed8a3672df654a445cef

**Comment by benjycui** [comment]
Updated.

**Comment by benjycui** [comment]
CI passed.

**Pull request merged**